core/api/src/config/schema.types.d.ts

```diff
@@ -53,9 +60,7 @@ type YamlSchema = {
   }
   bria: {
     hotWalletName: string
-    queueNames: {
-      fast: string
-    }
+    payoutQueues: PayoutQueuesConfigItem[]
     coldStorage: {
```

Comment:
probably is better to do a complete refactor here:

- coldWalletName instead of coldStorate.walletName
- hotToColdRebalanceQueueName at the same level of hotWalletPayoutQueues